ui/dashboards/src/components/Panel.tsx

```diff
@@ -52,20 +54,37 @@ export function Panel(props: PanelProps) {
     >
       <CardHeader
         title={
-          <Typography
-            component="h2"
-            variant="body2"
-            fontWeight={(theme) => theme.typography.fontWeightMedium}
-            whiteSpace="nowrap"
-            overflow="hidden"
-            textOverflow="ellipsis"
-          >
-            {definition.display.name}
-          </Typography>
+          <>
+            <Box
+              sx={{
+                display: 'flex',
+                alignItems: 'center',
+              }}
+            >
+              <Typography
+                component="h2"
+                variant="body2"
+                fontWeight={(theme) => theme.typography.fontWeightMedium}
+                whiteSpace="nowrap"
+                overflow="hidden"
+                textOverflow="ellipsis"
+              >
+                {definition.display.name}
+              </Typography>
+              {definition.display.description && (
+                <Tooltip description={definition.display.description} placement={TooltipPlacement.Right}>
+                  <InformationOutlineIcon
+                    sx={{ width: '16px', height: '16px', position: 'relative', left: '4px', cursor: 'pointer' }}
```

Comment:
same question here. Don't you want to make the size depending on the size of the screen ? Like should it be. the same size for a small screen or on a phone ?

I'm even wondering if on a phone screen, it makes sense to display the tooltip